### PR TITLE
Move background job registration to install update 8.1

### PR DESCRIPTION
**Pull request by nickvergessen**
@MorrisJobke @blizzz @icewind1991

### CODE REVIEW

**Comment by karlitschek** [comment]
good idea! :+1:

**Comment by icewind1991** [comment]
Doesn't ldap need a version increase?

File: lib/private/legacy/search/result.php [hunk 1]
```diff
@@ -39,3 +39,6 @@ public function __construct($id = null, $name = null, $link = null, $type = null
 		$this->type = $type;
 	}
 }
+
+OCP\Backgroundjob::registerJob('OCA\user_ldap\lib\Jobs');
```
Comment (MorrisJobke) [inline lib/private/legacy/search/result.php]: This looks fishy :fish:

Comment (nickvergessen) [inline lib/private/legacy/search/result.php]: Yes, I did not do that

**Comment by nickvergessen** [comment]
> Doesn't ldap need a version increase to?

The problem is that master is used the +0.0.1 version. we should really increment the second digit on a major release, so we don't run into problems like this in the future.

File: lib/private/legacy/search/result.php (hunk 1 above)
Comment (nickvergessen) [inline lib/private/legacy/search/result.php]: Rebased and fixed

**Comment by icewind1991** [comment]
:+1:

**Pull request merged**